tests/fixtures/contracts.json

```diff
@@ -0,0 +1,28 @@
+{
+  "ratePhezApp": {
```

Comment:
I like seeing this contract in here, but maybe should go for a more professional one :slightly_smiling_face:

Reply:
I define these as "soft" easter egg more than unprofessionnal one. 

In Wallet-lib, failure of generating a walletId is expressed with Squawk 7700 (which is the std emergency code you are supposed to put in your transponder if emergency, which is REAAAALLY not in line with it not being an easter egg). 

My take on this being, if it do the test with no downplay, and that it is not anything that is affecting negatively the feelings of another human being, then, I'm not sure we need. 
But that's a good question in the end 🤔 
Will let it here there, but yeah, may be your contract gonna be dropped :P